pkg/cli-runtime/emojidata.go

```diff
@@ -0,0 +1,19 @@
+package cli
+
+type Icon rune
+
+const (
+	FloppyDisk    Icon = '💾'
```

Comment:
package `cli` does not explain what the constants are, add a prefix `Icon` to each const i.e. `DeliveryIcon`

Reply:
These icons could be shared for different headers or messages, adding suffix could be misleading.